packages/jellyfish-crypto/__tests__/bs58.test.ts

```diff
@@ -0,0 +1,57 @@
+import { Bs58 } from '../src'
+
+const fixture = {
+  base58: 'dFFPENo7FPMJpDV6fUcfo4QfkZrfrV1Uf8',
+  prefix: 0x5a,
+  h160: '1410c734d66b986f2a7c2c340a1ee18d83d9b5a2',
+
+  invalidCharset: 'dFFPENo7FPMJpDV6fUcfo4QfkZrfrV1UfI',
+  invalidChecksum: 'dFFPENo7FPMJpDV6fUcfo4QfkZrfrV1Uf7'
```

Comment:
For these 2 you should move it to line 21 and 30 since you are using it once, it's easier to read.

I hate the use for fixtures because everyone starts to fall back on them. It becomes a central point of code failure/masses. 

> The idea is to treat testcode as real code, and writing a story for them.